src/utils/firebaseSnapshots.tsx

```diff
@@ -24,20 +24,37 @@ export const getEvents = async () => {
   }
 };
 
-export const addUser = async (userData: UserCredential, firstName: string | undefined, lastName: string | undefined, email: string | null, userId: string | undefined) => {
-    try {
-        const userDocRef = doc(db, "users", userData.user.uid);
-        
-        await setDoc(userDocRef, {
-        firstName: firstName,
-        lastName: lastName,
-        email: email,
-        UUID: userId,
-        events: [],
-      });
-      } catch (error: unknown) {
-        if (error instanceof FirebaseError) {
-          console.error(error.code)
-        }
-      }
-  }
\ No newline at end of file
+export const getUser = async (userId: string) => {
+  try {
+    const userCollectionRef = doc(db, "users", userId);
+    const data = await getDoc(userCollectionRef);
+    const currentUser = data.data();
+    return currentUser;
+  } catch (error: unknown) {
+    console.error(error);
+  }
+};
```

Comment:
can you move this underneath just for consistency , otherwise will cause issues when others try to merge.

Reply:
move it underneath addUser()?

Reply:
yep